versioned_docs/version-6.x.x/other-topics/dialect-specific-things.md

```diff
@@ -224,6 +224,15 @@ Sequelize also lets you pass credentials in URL format:
 const sequelize = new Sequelize('oracle://user:pass@hostname:port/servicename');
 ```
 
+You can pass an Easy Connect String, a Net Service Name, or a Connect Descriptor to the Sequelize constructor using `dialectOptions.connectString`:
+
+```js
+const sequelize = new Sequelize({dialect: 'oracle', username: 'user', password: 'password', dialectOptions: {connectString: 'inst1'}});
```

Comment:
```suggestion
const sequelize = new Sequelize({
    dialect: 'oracle',
    username: 'user',
    password: 'password',
    dialectOptions: {
        connectString: 'inst1'
     }
 });
```
Just for readability

Reply:
There is an identation issue in your suggestion (penultimate bracket), we also use 2 spaces in the rest of our codebase :)

Reply:
Good catch, fixed